README.md

```diff
@@ -205,6 +206,18 @@ the others wait their turn. This prevents coordination problems that can result
 in incorrect output files being produced. If `output` is set to an empty array,
 then this restriction is removed.
 
+## Extra arguments
+
+You can pass extra arguments to a Wireit script with _two_ double-dashes:
+
+```sh
+npm run build -- -- --verbose
+```
+
+The first double-dash tells `npm` to pass the remaining arguments to Wireit, and
+the second double-dash tells Wireit to pass the remaining arguments to the
+command.
```

Comment:
Mention that --verbose is only passed to the "build" script

Reply:
Done